Trappist/src/Promact.Trappist.Web/wwwroot/app/tests/test-view/test-view.component.ts

```diff
@@ -40,8 +40,10 @@ export class TestViewComponent implements OnInit {
     tooltipMessage: string;
     isCategorySelected: boolean;
     disablePreview: boolean;
+    duplicateTestDialogData: DuplicateTestDialogComponent;
+    deleteTestDialogData: DeleteTestDialogComponent;
 
-    constructor(public dialog: MdDialog, private testService: TestService, public snackBar: MdSnackBar, private router: Router, private route: ActivatedRoute) {
+    constructor(public dialog: MdDialog, private testService: TestService, public snackBar: MdSnackBar, private router: Router, public route: ActivatedRoute) {
```

Comment:
same

Reply:
It need not be public. Will have to change it.